package.json

```diff
@@ -4,10 +4,10 @@
   "description": "",
   "main": "index.js",
   "watch": {
-    "start": "{src}/**/*.js"
+    "start": "src/**/*.js"
   },
   "scripts": {
-    "start": "npm run eslint && webpack-dev-server --mode development --open --hot",
+    "start": "npm run eslint && webpack-dev-server --mode development --hot",
```

Comment:
The `--open` has been removed to prevent the browser from opening a new tab on every file change

Reply:
okay